File: mambo-protocol/src/main/scala/org/mambo/protocol/enums/ServerStatusEnum.scala




















// Generated on 05/02/2013 20:27:17
// Scala version by Blackrush
package org.mambo.protocol.enums

import org.mambo.protocol.Enumeration

object ServerStatusEnum extends Enumeration {
  val statusUnknown = Value(0, "statusUnknown")
  val offline = Value(1, "offline")
  val starting = Value(2, "starting")
  val online = Value(3, "online")
  val nojoin = Value(4, "nojoin")
  val saving = Value(5, "saving")
  val stoping = Value(6, "stoping")
  val full = Value(7, "full")
}

File: mambo-protocol/src/main/java/org/mambo/protocol/enums/ServerStatusEnum.java


// Generated on 05/08/2013 19:04:45
package org.mambo.protocol.enums;

public enum ServerStatusEnum {
    STATUS_UNKNOWN(0),
    OFFLINE(1),
    STARTING(2),
    ONLINE(3),
    NOJOIN(4),
    SAVING(5),
    STOPING(6),
    FULL(7);
    
    private int value;
    ServerStatusEnum(int value) { this.value = value; }
    public int value() { return value; }
    
    public static ServerStatusEnum valueOf(int value) {
    	for (ServerStatusEnum a : values()) {
    		if (a.value == value)
    			return a;
    	}
    	return null;
    }
    
}
